### DISCUSSION TITLE
Error in function 'export_ctf.m'

### DISCUSSION THREAD

**Discussion by eugenianietov** [mtex-toolbox/mtex#615] [Ask Anything]
Hi everyone,

I was trying to export a modified EBSD to a .ctf file with MTEX 5.5, and I was getting an error on line 259 of the function 'export_ctf.m'. I looked into it and I easily solved it by replacing 'ebsdList.y' by 'ebsdList.prop.y' as follows.
`[~,ind] = sort(ebsdList.prop.y);`
I think that this error might also be present in other functions such as 'export.m'.  I hope it helps!!

I don't know if this topic should go in another category such as 'Announcements', I have never 'solved' anything related to MTEX haha. Anyway, I hope this helps the developers at least!

Cheers,

Eugenia

**Comment by ralfHielscher** [comment]
Hi Eugenia,

thank you reporting this issue. We will include your fixed into the next release. The "correct" place would have been a bug report:  https://github.com/mtex-toolbox/mtex/issues/new/choose  :)

All the best,

Ralf.
